bodhi/server/consumers/masher.py

```diff
@@ -958,12 +958,24 @@ def compose_atomic_trees(self):
                 release['repos']['updates'] = mash_path
 
             # Compose the tree, and raise an exception upon failure
+            notifications.publish(topic="ostree.compose.start",
+                                  msg=dict(tag=tag),
+                                  force=True)
             result = composer.compose(release)
             if result['result'] != 'success':
                 self.log.error(result)
+                notifications.publish(topic="ostree.compose.fail",
+                                      msg=dict(tag=tag),
+                                      force=True)
                 raise Exception('%s atomic compose failed' % tag)
             else:
                 self.log.info('%s atomic tree compose successful', tag)
+                notifications.publish(
+                    topic="ostree.compose.finish",
+                    msg=dict(tag=tag,
+                             ref=result.get('ref'),
+                             commitid=results.get('commitid')),
+                     force=True)
```

Comment:
It looks like this ```force``` isn't lined up with the ```msg``` above it, maybe one too many spaces?